src/workerskv/mod.rs

```diff
@@ -0,0 +1,38 @@
+use crate::response::APIResult;
+use chrono::offset::Utc;
+use chrono::DateTime;
+
+mod create_namespace;
+mod list_namespace_keys;
+mod list_namespaces;
+mod remove_namespace;
+mod rename_namespace;
```

Comment:
Discussed offline: these things aren't `pub` but should be, so callers can construct the relevant Params objects, etc.